woocommerce/woocommerce
# Package Release: Update usage of NPM auth token

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





NPM auth token was being incorrectly used, this PR fixes it according to [NPM docs](https://docs.npmjs.com/using-private-packages-in-a-ci-cd-workflow#create-and-check-in-a-project-specific-npmrc-file).



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. I kicked off the Release workflow on a package that was ready to publish using this branch. See the workflow succeed here: https://github.com/woocommerce/woocommerce/actions/runs/7722378461/job/21050460230
2. Expand the "Execute script" and confirm the script that was executed referencing this branch (`fix/npm-auth-token-use`) and the `@woocommerce/create-woo-extension` package.
3. Note also the `1.1.0` version successfully published at the end of the section's logs
4. Confirm the package was indeed updated on NPM: https://www.npmjs.com/package/@woocommerce/create-woo-extension



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
